linera-core/src/client.rs

```diff
@@ -1321,7 +1322,26 @@ where
     }
 
     /// Creates an application by instantiating some bytecode.
-    pub async fn create_application(
+    pub async fn create_application<A: ContractAbi>(
```

Comment:
so this is only used in tests where the type is known at compile time? If so would it not make sense to `cfg(test)` it?

Reply:
In principle, someone could write their client with a specific application in mind, I suppose?